generated/nirfsg/nirfsg/session.py

```diff
@@ -6663,10 +6463,11 @@ def check_if_script_exists(self, script_name):
         script_exists = self._interpreter.check_if_script_exists(script_name)
         return script_exists
 
+    @ivi_synchronized
     def check_if_waveform_exists(self, waveform_name):
         r'''check_if_waveform_exists
 
-        Returns whether the waveform that you specify as **WAVEFORM_NAME** exists.
+        Returns whether the waveform that you specify as WAVEFORM_NAME exists.
```

Comment:
Another thing to fix in a future PR when you comb over things.
I think what's happening here is that WAVEFORM_NAME isn't the parameter name in C, so it's not getting replaced.

Reply:
No, the C parameter name is waveformName.